src/common/pico_time/time.c

```diff
@@ -297,6 +297,11 @@ static int64_t sev_callback(__unused alarm_id_t id, __unused void *user_data) {
 #endif
 
 void sleep_until(absolute_time_t t) {
+#if PICO_ON_DEVICE && !PICO_ALLOW_SLEEP_IN_EXCEPTION && !defined(NDEBUG)
```

Comment:
So `PICO_ALLOW_SLEEP_IN_EXCEPTION=1` just removes the `panic()`?  This only seems useful in combination with `PICO_TIME_DEFAULT_ALARM_POOL_DISABLED=1`, since otherwise you still die when you sleep in an exception, just quietly. Is it worth `#error`ing on invalid combinations of those flags?

Reply:
ah yes, the intention was for sleep to work in that case (as a busy_wait) but in all fairness if the caller is going to go to that trouble they should call busy_wait themselves!